
# This is a message to hold data with position and trajectory information
float64 s
float64 ey
float64 epsi
float64 v
float64 s_start
float64[] coeffX
float64[] coeffY
float64[] coeffTheta
float64[] coeffCurvature
